test/types/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "module",
+    "dependencies": {
+        "pdfjs-dist": "../../build/dist/"
+    }
+}
```

Comment:
This is lacking a new-line at the end of the code, and can we please use *two spaces* of indentation instead?